### PR TITLE
Release Notes 4.2 Copy-edit

### PR DESCRIPTION
**Pull request by elstoc**

### CODE REVIEW

**Review by dterrahe — COMMENTED** [review]
Some suggestions/clarifications

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -31,270 +31,293 @@ Since darktable 4.0:
...
 
-- JPEG XL (read / write)
+- darktable is now able to read and write JPEG XL images
 
-- Keep modules fully visible when opening or expanding and move with
-  a smooth transition effect. The effect can be sped up or switched
-  off with preferences/miscellaneous/duration of ui transitions.
+- A new animation effect has been added when expanding/collapsing modules.
+  As part of this change, when you expand a module it will be automatically
+  scrolled to ensure the entire module is visible on-screen.
```
[the remark this answers is not in the document]
Comment (elstoc) [reply RELEASE_NOTES.md:79]: Good point

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -31,270 +31,293 @@ Since darktable 4.0:
...
+  (for tablet users). The mask manager has been enhanced with a new collapsible
   section containing controls for changing masks properties.
 
-  It is also possible to change a properly shared by a group of mask.
+  It is also possible to change a property that is shared by a group of masks.
 
-  At the same time the circle and ellipse masks have been enhanced
-  with new on-canvas controls to change the size and feather.
+  The circle and ellipse masks have also been enhanced with new on-canvas
+  controls to change the mask size and feathering.
```
[the remark this answers is not in the document]
Comment (elstoc) [reply RELEASE_NOTES.md:115]: agreed

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -31,270 +31,293 @@ Since darktable 4.0:
...
 
-- Remove the automatic check for OpenCL headroom as this was causing
-  more problems than it solve.
+- Remove the automatic check for OpenCL headroom - it was causing
+  more problems than it solved.
 
-- Fix non blinking cursor in the processing module search dialog. This
-  was making the module looks like it was not active.
+- Fix non-blinking cursor in the darkroom search dialog. This
+  was making the dialog look as if it was inactive.
```
[the remark this answers is not in the document]
Comment (elstoc) [reply RELEASE_NOTES.md:272]: yeah, I can't come up with anything better than "bar" but agreed, dialog is wrong

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -31,270 +31,293 @@ Since darktable 4.0:
...
-  even works if those options are changed from another widget in the qap.
+- In the quick access panel, parameters are now hidden if they are also hidden
+  in the full module (for those parameters that conditionally display based on
+  the state of other parameters). This even works if those parameters are
+  changed from another widget in the quick access panel.
 
 - Slightly improve the blue color label for better visibility in the
   dark theme.
 
 - Fix navigation window zoom indicator label display. It was sometime
```
[the remark this answers is not in the document]
Comment (elstoc) [reply RELEASE_NOTES.md:308]: fixed

**Review by TurboGit — APPROVED** [review]
Thanks!

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -31,270 +31,296 @@ Since darktable 4.0:
...
+  pixelpipe functionality. This means that it can now be zoomed and panned
+  with keyboard/mouse.
+
+- The duplicate manager previously used a different pipe routine to calculate
+  its previews (i.e. when long-pressing on a duplicate image thumbnail) which
+  often meant that the displayed duplicates differed from the main darkroom
+  view in subtle ways. Use of the new pipe routine now means that these previews
+  will be identical to those produced during darkroom editing.
+
+- It is now possible to preview the affect of a user-generated style on
```
Comment (ralfbrown) [inline RELEASE_NOTES.md:64]: effect

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -31,270 +31,296 @@ Since darktable 4.0:
...
+  it, if present.
 
-- The reading of 16-bit half float for TIFF format is now done using
+- Reading of 16-bit half float for the TIFF format is now undertaken using
   the Imath library.
 
-- Enable actions on the export format and storage widgets. This means
-  that they can be mapped to shortcut.
+- The format and storage parameters in the export module can now be mapped to
+  to shortcuts.
```
Comment (ralfbrown) [inline RELEASE_NOTES.md:164]: to to -> to

File: RELEASE_NOTES.md [hunk 7]
```diff
@@ -31,270 +31,296 @@ Since darktable 4.0:
...
-  use CPU tiling if there is not enough memory on the OpenCL card and
-  so would require a lot of tiles to be handled on the card. At the
-  end, the large number of tiles plus the overlapping area will make
+- Introduced balanced "OpenCL vs CPU" tiling. This makes it possible to
+  use CPU tiling if there is not enough memory on the OpenCL card
+  (which would require a lot of tiles to be handled on the card). In the
+  end, the large number of tiles plus the overlapping area would make
   the use of the OpenCL code patch slower (or much slower) than
-  handling the image without tiling on CPU.
+  handling the image without tiling on the CPU.
```
Comment (ralfbrown) [inline RELEASE_NOTES.md:171]: patch -> path

File: RELEASE_NOTES.md (hunk 5 above)
Comment (TurboGit) [reply RELEASE_NOTES.md:64]: fixed.

File: RELEASE_NOTES.md (hunk 6 above)
Comment (TurboGit) [reply RELEASE_NOTES.md:164]: fixed.

**Pull request merged**